completions/fish/podman.fish

```diff
@@ -0,0 +1,3201 @@
+
+function __fish_podman_no_subcommand --description 'Test if podman has yet to be given the subcommand'
+    for i in (commandline -opc)
+        if contains -- $i attach build commit container cp create diff events exec export generate generate-completion healthcheck history image images import info inspect kill load login logout logs mount pause play pod port ps pull push restart rm rmi run save search start stats stop system tag top umount unpause varlink version volume wait
```

Comment:
maybe I've missed something, but is 'generate-completion' valid?

Reply:
Oh, I think I see now, it's for the shell completions itself, not necessarily a podman command.

Reply:
Yeah, sorry I forgot to remove it after my modification of podman :+1: